docker/test/fasttest/Dockerfile

```diff
@@ -20,10 +20,19 @@ RUN apt-get update \
         pv \
         jq \
         zstd \
+        sudo \
     --yes --no-install-recommends
 
 RUN pip3 install numpy scipy pandas Jinja2
 
+RUN export CODENAME="$(lsb_release --codename --short | tr 'A-Z' 'a-z')" \
+    && apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF \
+    && echo "deb https://download.mono-project.com/repo/ubuntu stable-focal main" | sudo tee /etc/apt/sources.list.d/mono-official-stable.list \
```

Comment:
This is a supply chain vulnerability (cannot be accepted), but let's see what will fail next.